gulpfile.mjs

```diff
@@ -14,8 +14,10 @@ import { npmScriptTask } from './tasks/run.mjs'
  * Runs JavaScript code quality checks, documentation, compilation
  */
 gulp.task('scripts', gulp.series(
-  npmScriptTask('lint:js'),
-  compileJavaScripts,
+  gulp.parallel(
+    npmScriptTask('lint:js'),
+    compileJavaScripts
+  ),
```

Comment:
**question** Won't that mangle the output if anything logs? Feels like a neat little speed up though 🙌🏻

Reply:
Output is nice actually

For "linting only" messages everything is logged as it was but compilation continues
For invalid code you'll see the linting message AND a short Rollup or Sass error

It's mainly to remove an annoyance:

1. Open a component `*.mjs` or `*.scss` file
2. Update or add some new code
3. Save the changes… ⏳ 
4. **Linting error on invalid JSDoc or SassDoc etc** 🚨

It felt wrong to block developers from compiling valid code unless it 100% matched the style rules

Reply:
Like commenting out some code to do a quick browser refresh but being blocked on

>**'normaliseDataset' is defined but never used.** eslint([no-unused-vars](https://eslint.org/docs/rules/no-unused-vars))

Reply:
Now you see:

```console
[11:59:20] Starting 'compile:js'...
[11:59:20] Finished 'compile:js' after 86 ms

/path/to/project/govuk-frontend/src/govuk/components/accordion/accordion.mjs
  2:10  error  'normaliseDataset' is defined but never used  no-unused-vars

✖ 1 problem (1 error, 0 warnings)
```

☝️ `Finished 'compile:js'` 🎉

Reply:
That's brilliant then! Super neat! 🙌🏻

Reply:
Moved to:

* https://github.com/alphagov/govuk-frontend/pull/3127